frontend/src/components/Item/RelatedItems.vue

```diff
@@ -1,49 +1,47 @@
 <template>
-  <div v-if="relatedItems.length > 0">
-    <div v-if="!vertical" class="related-items">
-      <swiper-section
-        :title="$t('youMayAlsoLike')"
-        :items="relatedItems"
-        :loading="loading" />
-    </div>
-    <div v-else-if="vertical">
-      <h2 v-if="!loading && relatedItems.length > 0" class="text-h6 text-sm-h5">
-        <slot>
-          {{ $t('youMayAlsoLike') }}
-        </slot>
-      </h2>
-      <!-- TODO: Wait for Vuetify 3 implementation (https://github.com/vuetifyjs/vuetify/issues/13504) -->
-      <!-- <v-skeleton-loader v-else-if="loading" type="heading" /> -->
-      <v-list bg-color="transparent" lines="two">
-        <div v-if="!loading && relatedItems.length > 0">
-          <v-list-item
-            v-for="relatedItem in relatedItems"
-            :key="relatedItem.Id"
-            :to="getItemDetailsLink(relatedItem)">
-            <template #prepend>
-              <v-avatar>
-                <v-avatar color="card">
-                  <blurhash-image :item="relatedItem" />
-                </v-avatar>
+  <div v-if="!vertical">
+    <swiper-section
+      :title="t('youMayAlsoLike')"
+      :items="relatedItems"
+      :loading="loading" />
+  </div>
+  <div v-else-if="vertical">
+    <h2 v-if="!loading && relatedItems.length > 0" class="text-h6 text-sm-h5">
+      <slot>
+        {{ t('youMayAlsoLike') }}
+      </slot>
+    </h2>
+    <!-- TODO: Wait for Vuetify 3 implementation (https://github.com/vuetifyjs/vuetify/issues/13504) -->
+    <!-- <v-skeleton-loader v-else-if="loading" type="heading" /> -->
+    <v-list bg-color="transparent" lines="two">
+      <div v-if="!loading && relatedItems.length > 0">
+        <v-list-item
+          v-for="relatedItem in relatedItems"
+          :key="relatedItem.Id"
+          :to="getItemDetailsLink(relatedItem)">
```

Comment:
```suggestion
        <v-list-item
          v-for="relatedItem in relatedItems"
          :key="relatedItem.Id"
          :to="getItemDetailsLink(relatedItem)"
          :title="relatedItem.Name"
          :subtitle="relatedItem.ProductionYear">
```

to free up the HTML below